### DISCUSSION TITLE
loader.js 오류

### DISCUSSION THREAD

**Discussion by suuu111** [storycraft/node-kakao#431] [Q&A]
`internal/modules/cjs/loader.js:419
    throw e;
    ^

Error: Cannot find module 'C:\새 폴더 (2)\dist\index.js'`

이런 오류가 뜨는 이유가 있을까요? package.json 확인 후 파일 있는지 확인했지만 다 있는데 혹시 수정 방법 알수있을까요?

**Comment by generalxd** [comment]
https://selfish-developer.com/entry/error-internalmodulescjsloaderjs883

**Reply by generalxd** [reply]
아니면 노드JS 최신버전으로 업데이트 해보세요

**Reply by suuu111** [reply]
노드JS 재설치 및 저 방법으로 노드모듈을 날리고 캐시 삭제후 다시 시도해봤지만 똑같은 문제가 뜹니다...ㅠㅠㅠ

**Reply by generalxd** [reply]
우분투 사용하시나요?

그렇다면 노드 최신버전 설치를 그냥하시면안됩니다.

**Reply by suuu111** [reply]
아뇽 우분투 사용 안하고 있어영

**Reply by generalxd** [reply]
저거 지금보니 한글경로라 그런거아닐까요
